### PR TITLE
TapToPlace anchor manager check

### LINKED ISSUE #636: TapToPlace script Small Issue
Have a feeling that we should check if the anchorManager.AnchorStore exists and are ready before Attach or Remove Anchors in the OnInputClicked handler.
Something like:
if (anchorManager.AnchorStore != null)
{
// Add world anchor when object placement is done.
anchorManager.AttachAnchor(gameObject, SavedAnchorFriendlyName);
}

### PR DESCRIPTION
**Pull request by jessemcculloch**
Adds null check on Anchor Manager before trying to add anchor

Resolves #636

### CODE REVIEW

**Comment by jessemcculloch** [comment]
Ok, I moved the entire check to wrap the if-else statement, and log an error if anchor manager does not exist.

File: Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs [hunk 1]
```diff
@@ -132,21 +132,29 @@ public virtual void OnInputClicked(InputClickedEventData eventData)
             // On each tap gesture, toggle whether the user is in placing mode.
             IsBeingPlaced = !IsBeingPlaced;
```
Comment (thebanjomatic) [inline Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs]: Would we still want to toggle the "IsBeingPlaced" value in this case, or should we just do nothing if the AnchorStore isn't available yet?

**Comment by StephenHodgson** [comment]
@jessemcculloch, any chance you could address Adams concerns?

**Comment by jessemcculloch** [comment]
Yes, but not till next week.  I'm at Build this week.

**Comment by jessemcculloch** [comment]
Based on the research above, it sounds like the changes in this PR are not needed.  Closing

**Pull request closed without merging**